Makefile

```diff
@@ -1,4 +1,5 @@
 # Makefile
+# swig -tcl -v -c++ -cppext cpp openfpg.i
```

Comment:
We need to build from source always, so the swig step has to be part of the Makefile (Better in cmake) and regenerate the swig wrapper on demand of if the .h changes (Incremental build).